package.json

```diff
@@ -16,7 +16,7 @@
         "@ironcorelabs/ironweb": "^2.0.1",
         "angular-in-memory-web-api": "^0.6.1",
         "core-js": "^2.4.1",
-        "es6-promise": "^4.2.6",
+        "es6-promise": "^4.2.8",
```

Comment:
Was this change to hope and fix the security vulnerability on lodash in this repo? From what I can tell lodash isn't a dependency of `es6-promise` so I'm not sure this is going to fix anything.

Reply:
I did a `npm audit fix`, and it made that change, then updated a bunch of downstream dependencies.

Something updated `lodash` from `4.17.11` to `4.17.14`, which should resolve the security issue. But if that isn't safe to do, I can nuke the PR.

Reply:
I really have no idea if it's safe to do or not. I've never tried to update specific sub dependencies like that. I think the only way to truly know is to start this thing up and run through it to see if it still works.